DataFormats/HcalDetId/interface/HcalTestNumbering.h

```diff
@@ -6,6 +6,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 #include <boost/cstdint.hpp>
+#include <stdint.h>
```

Comment:
usually we recommend using `<c*>` over `<*.h>` headers for STL.

In addition, this file already includes `<boost/cstdint.hpp>`, which seems to define `uint32_t`. I am confused why the specific added include is necessary.

Reply:
@kpedro88 , when CXX Modules are generated (using clang) then clang complains about unknown type `uint32_t`

Reply:
@smuzaffar ok, but is this behaviour related to the use of <*.h> vs <c*> ? I would naively expect it is not...

Reply:
@smuzaffar , we have in many other places

#include <cstdint>

likely it is fully equivalent, do we have a guideline what to use?

Reply:
I will try cstdint and update the PR if works

Reply:
@civanch I do not think we have any official written guideline, I think this is more a habit, @smuzaffar @davidlange6 do you confirm?

Reply:
fixed, it was enough to releance boost/cstdint with csdtint